openshift/4.0/templates/sonarqube/sonarqube-postgresql-template.yaml

```diff
@@ -1,404 +0,0 @@
-apiVersion: v1
```

Comment:
Template for deploying SonarQube.

Created:
	Service: sonarqube
	Route: sonarqube
	Secret: sonarqube-admin-password
	Secret: postgresql-sonar-password
	ImageStream: sonarqube
	DeploymentConfig: sonarqube
	Service: postgresql-sonarqube
	DeploymentConfig: postgresql-sonarqube
	PersistentVolumeClaim: postgresql-sonarqube-data
	PersistentVolumeClaim: sonarqube-data
	NetworkSecurityPolicy: pod-to-pod-permit-${NAME}-${SOURCE_ROLE}-${DESTINATION_ROLE}